### DISCUSSION TITLE
Pull lux formula into homebrew-core

### DISCUSSION THREAD

**Discussion by jamesharr** [Homebrew/discussions#62] [Everyday usage]
I approached the author of the Lux tap to pull that formula into the main homebrew-core repo. When hawk had originally approached homebrew to do this, the project wasn't popular enough to do so. I believe it's now reached the point of being popular enough.

Is there anything else we need to do before creating a branch + PR to pull into the main repo?

Related discussion thread in the tap repo: https://github.com/hawk/homebrew-hawk/issues/1

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
